econet_base.yaml

```diff
@@ -54,6 +54,13 @@ econet:
   update_interval: ${econet_update_interval}
 
 sensor:
+  - platform: econet
+    name: "Active Alerts"
+    sensor_datapoint: ALRMALRT
+    accuracy_decimals: 0
+    icon: "mdi:water-boiler-alert"
+    state_class: "measurement"
+    entity_category: "diagnostic"
```

Comment:
Did you also see the individual ALMCODE1? What format were they in? Any reason not to include those sensors here as welll?